bin/sozo/src/commands/options/transaction.rs

```diff
@@ -37,6 +37,12 @@ pub struct TransactionOptions {
     )]
     #[arg(global = true)]
     pub receipt: bool,
+
+    #[arg(long)]
+    #[arg(help = "If --wait is set, the link to debug the transaction with Walnut is displayed \
+                  on stdout. Otherwise, it is a no-op.")]
+    #[arg(global = true)]
+    pub walnut: bool,
```

Comment:
i think we should make setting `--walnut` also implies using `--wait` without having to specify it manually.